COMMON_ISSUES.md

```diff
@@ -9,23 +9,43 @@ before opening a new issue. We recommend regularly upgrading to the latest versi
 <details>
 <summary>Chrome memory leak might cause degraded experience in group rooms</summary>
 <p>
-  
+
    Chrome has a memory leak, which is most apparent in long running Group Rooms with 30+ Participants with most of them share media. For details, go [here](https://github.com/twilio/twilio-video.js/issues/1449).
 </p>
 </details>
 
+<details>
+<summary>With opus dtx enabled, sometimes background noise is heard on muted AudioTracks</summary>
+<p>
+
+   twilio-video.js enables DTX (discontinuous transmission) by default for opus. Webrtc has an issue which sometimes causes background noise to be heard on muted track when DTX is enabled, For details, see [here](https://bugs.chromium.org/p/webrtc/issues/detail?id=13051).
```

Comment:
It was merged into: https://bugs.chromium.org/p/webrtc/issues/detail?id=12790

Reply:
yep, but that one is too technical, the duplicated one has more context.